### PR TITLE
Update COMPILING.md

### PR DESCRIPTION
**Pull request by RobyDati**
Updated point 5. of ## iOS  section with more details on Xcode settings in order to get the Jamulus app installed on iPhone device. Added the following details: "For instance, you can use your free Apple Developer Program Account as a Personal Team, and set it up under Xcode Menu->Preferences->Accounts. Then choose a Bundle Identifier at your choice in the relevant field under General Tab in section Identity. Then go to Signing & Capabilities tab, section Identity, the Automatic manage signing option and you should get Team: (Your Name) (Personal Team), Bundle identifier: (the same you modified on General Tab), Provisioning Profile: Xcode Management Profile, Signing Certificate: Apple Development (your e-mail used for signing in to Apple)"

**Short description of changes**

**Context: Fixes an issue?**

**Does this change need documentation? What needs to be documented and how?**

**Status of this Pull Request**

**What is missing until this pull request can be merged?**

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I tested my code and it does what I want
- [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I've filled all the content above

### CODE REVIEW

**Review by ann0see — COMMENTED** [review]
Great! Thank you very much for updating this file. 

GitHub can be very confusing so don’t panic 😀. In general it looks good. There might still be some small changes needed for the styling (add more bullet points to give the text more structure)? But if you don’t want to update it yourself it’s ok. 

Mainly the file should have you as contributor listed to know who edited what and when.

File: COMPILING.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ Will build the file and make it available in `./Release/Jamulus.app`
...
 3. Go to the folder of the Jamulus source code via terminal and run `/path/to/qt/5.15.2/ios/bin/qmake -spec macx-xcode Jamulus.pro` to generate an .xcodeproject file
 4. Open the generated .xcodeproject in Xcode
-5. Go to the Signing & Capabilities tab and fix signing errors by setting a team. Xcode might tell you if you need to change anything else like the bundle id.
+5. Go to the Signing & Capabilities tab and fix signing errors by setting a team. Xcode might tell you if you need to change anything else like the bundle id. For instance, you can use your free Apple Developer Program Account as a Personal Team, and set it up under Xcode Menu->Preferences->Accounts. Then choose a Bundle Identifier at your choice in the relevant field under General Tab in section Identity. Then go to Signing & Capabilities tab, section Identity, the Automatic manage signing option and you should get Team: (Your Name) (Personal Team), Bundle identifier: (the same you modified on General Tab), Provisioning Profile: Xcode Management Profile, Signing Certificate: Apple Development (your e-mail used for signing in to Apple)
```
Comment (ann0see) [inline COMPILING.md:149]:
```suggestion
5. Go to the Signing & Capabilities tab and fix signing errors by setting a team. Xcode will tell you what you need to change.
* If have a free Apple Developer Account, you can use it as a "Personal Team":
* Set it up under Xcode Menu->Preferences->Accounts. 
* Then choose a Bundle Identifier at your choice in the relevant field in the "General" Tab (in section "Identity")
* Now click on the "Signing & Capabilities" tab. In the section "Identity", the "Automatic manage signing" option should be selected.
* You should now see Team: (Your Name) (Personal Team), Bundle identifier: (the same you modified on General Tab), Provisioning Profile: Xcode Management Profile, Signing Certificate: Apple Development (your e-mail used for signing in to Apple)
```

**Review by ann0see — COMMENTED** [review]
Just a small comment. Will try to fire up my macOS environment.

File: COMPILING.md (hunk 1 above)
Comment (RobyDati) [reply COMPILING.md:149]: Thank you, suggested change are ok to me.

Comment (ann0see) [reply COMPILING.md]: Great.

**Review by ann0see — APPROVED** [review]
Should be squash merged

**Comment by hoffie** [comment]
Thanks, @RobyDati! Do you want to be added to the list of contributors within Jamulus?
You can do this yourself as part of this (see [`src/util.cpp`](https://github.com/RobyDati/jamulus/blob/5cd09b6365f4d068e7a4ea58afeca11c33cd8997/src/util.cpp#L492)) or we can do it, if you want?

CHANGELOG: Docs: iOS signing instructions have been added to the COMPILING.md documentation

**Comment by RobyDati** [comment]
Thank you Christian,

OK I wish to be added to the list, just in case I hope I may contribute more in the future. I am not sure I can do it myself, if you can do it for me I'll be grateful.

Best regards

Roberto

**Review by hoffie — APPROVED** [review]
Alright, I have pushed an [update](https://github.com/jamulussoftware/jamulus/pull/2315/commits/3be8d4e8b4a63854f07dbcab4d944ea8dff41eeb) to this PR and will merge on CI green.

**Pull request merged**